controllers/users/updateAvatar.js

```diff
@@ -0,0 +1,36 @@
+const { Unauthorized } = require("http-errors");
+const Jimp = require("jimp");
+const { User } = require("../../models");
+
+const path = require("path");
+const fs = require("fs/promises");
+
+const avatarDir = path.join(__dirname, "../../", "public", "avatars");
+
+const updateAvatar = async (req, res, next) => {
+  const { path: tempUpload, originalname } = req.file;
+  const { _id: id } = req.user;
+  const imageName = `${id}_${originalname}`;
```

Comment:
What happens if a user downloads the same file multiple times?

Reply:
nothing happens

Reply:
It's incorrect. It may be another file with same name and you must rewrite this file.

Reply:
I am not going to make any improvements or changes in this homework